index.d.ts

```diff
@@ -135,6 +135,29 @@ export type Options = {
 	string
 	>;
 
+	/**
+	Color of the background of the border.
+	*/
+	readonly borderBackgroundColor?: LiteralUnion<
+	| 'black'
+	| 'red'
+	| 'green'
+	| 'yellow'
+	| 'blue'
+	| 'magenta'
+	| 'cyan'
+	| 'white'
+	| 'blackBright'
+	| 'redBright'
+	| 'greenBright'
+	| 'yellowBright'
+	| 'blueBright'
+	| 'magentaBright'
+	| 'cyanBright'
+	| 'whiteBright',
+	string
+	>;
```

Comment:
This is used in multiple places. Move it to an exported `Color` type.

Reply:
Good idea.